darshan_logs/large_log/README.md

```diff
@@ -0,0 +1,11 @@
+This example is an anonymized version of the largest Darshan log generated
+on the Theta system at the ALCF in the month of October 2021.
+
+The log has been anonymized, converted to bzip2 compression, and is hosted
+as an external download.  It can be downloaded from the link listed in the
+theta-large-anonymized-example-2021-bz2.darshan.link file in this
+subdirectory.  After downloading, rename the resulting file to
+theta-large-anonymized-example-2021-bz2.darshan.
+
+You can also download all such large files in this repository in bulk using
+the `download-large-logs.sh` script at the top level of this repository.
```

Comment:
perhaps worth noting that this will automatically set the correct path/name for the log file as well

Reply:
Thanks, updated just now.